src/components/map/EventLayer.js

```diff
@@ -136,17 +150,23 @@ class EventLayer extends Layer {
         if (Array.isArray(data.rows)) {
             data.rows.forEach(row => {
                 const extent = row[header.extent].match(/([-\d.]+)/g);
+                const count = parseInt(row[header.count], 10);
+                const clusterId = ++this.clusterCount;
 
                 features.push({
                     type: 'Feature',
-                    id: row[header.points],
+                    id: clusterId,
                     geometry: JSON.parse(row[header.center]),
                     properties: {
-                        count: parseInt(row[header.count], 10),
+                        cluster: count > 1,
+                        cluster_id: clusterId,
+                        point_count: count,
+                        point_count_abbreviated: formatCount(count),
                         bounds: [
-                            [extent[1], extent[0]],
-                            [extent[3], extent[2]],
+                            [extent[0], extent[1]],
+                            [extent[2], extent[3]],
                         ],
+                        id: row[header.points],
```

Comment:
Doesn't this need to be `[EVENT_ID_FIELD]: row[header.points]`?  Or am I missing something?

Reply:
I don't think so as this is server cluster that don't make sense to download. I think you added [EVENT_ID_FIELD] for download purposes.

Reply:
Maybe true, but in that case I think the below `feature.properties.id || feature.properties[EVENT_ID_FIELD];` might break data download, since we're now always setting `.id`?  I don't remember how this logic worked off the top of my head, can look into it if you want.

Reply:
id is only set for server clusters, which we don't download - when the user download a server cluster layer we make new api requests and fetches all the events. I've tested that download still works like before, but feel free to do some additional testing :-)